src/dataflow/src/logging/materialized.rs

```diff
@@ -323,7 +324,7 @@ pub fn construct<A: Allocate>(
         let peek_current = peek.as_collection().map({
             move |(peek, worker)| {
                 Row::pack_slice(&[
-                    Datum::String(&format!("{}", peek.conn_id)),
+                    Datum::String(&format!("{}", peek.uuid)),
```

Comment:
I was going to say, drat, we need to update the docs on the format of this ID but .. well would you look at that! https://materialize.com/docs/sql/system-catalog/#mz_peek_active

Mind fixing up that doc? Also I think we should take the opportunity to change the name of the `uuid` column to `id` and give it type `uuid`, and change the name of the `id` column to `index_id`.

Reply:
Oh I only meant the catalog table. But if you think there are other things that should be renamed for clarity, don't let me stop you!